core/superchain.go

```diff
@@ -27,19 +27,21 @@ func LoadOPStackGenesis(chainID uint64) (*Genesis, error) {
 	}
 
 	genesis := &Genesis{
-		Config:     cfg,
-		Nonce:      gen.Nonce,
-		Timestamp:  gen.Timestamp,
-		ExtraData:  gen.ExtraData,
-		GasLimit:   gen.GasLimit,
-		Difficulty: (*big.Int)(gen.Difficulty),
-		Mixhash:    common.Hash(gen.Mixhash),
-		Coinbase:   common.Address(gen.Coinbase),
-		Alloc:      make(GenesisAlloc),
-		Number:     gen.Number,
-		GasUsed:    gen.GasUsed,
-		ParentHash: common.Hash(gen.ParentHash),
-		BaseFee:    (*big.Int)(gen.BaseFee),
+		Config:        cfg,
+		Nonce:         gen.Nonce,
+		Timestamp:     gen.Timestamp,
+		ExtraData:     gen.ExtraData,
+		GasLimit:      gen.GasLimit,
+		Difficulty:    (*big.Int)(gen.Difficulty),
+		Mixhash:       common.Hash(gen.Mixhash),
+		Coinbase:      common.Address(gen.Coinbase),
+		Alloc:         make(GenesisAlloc),
+		Number:        gen.Number,
+		GasUsed:       gen.GasUsed,
+		ParentHash:    common.Hash(gen.ParentHash),
+		BaseFee:       (*big.Int)(gen.BaseFee),
+		ExcessBlobGas: gen.ExcessBlobGas,
+		BlobGasUsed:   gen.BlobGasUsed,
```

Comment:
This literal is part of the validation flow for genesis checks for data in the superchain-registry. The genesis was being rebuilt incorrectly due to missing fields.